test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -324,6 +325,31 @@ TEST_P(ExtProcIntegrationTest, GetAndSetHeadersOnResponse) {
   EXPECT_THAT(response->headers(), SingleHeaderValueIs("x-response-processed", "1"));
 }
 
+// Test the filter using a configuration that uses the processing mode to
+// only send the response_headers message.
+TEST_P(ExtProcIntegrationTest, ProcessingModeResponseOnly) {
+  proto_config_.mutable_processing_mode()->set_request_header_mode(ProcessingMode::SKIP);
+  initializeConfig();
+  HttpIntegrationTest::initialize();
+  auto response = sendDownstreamRequest(nullptr);
+  handleUpstreamRequest();
+
+  ProcessingRequest response_headers_msg;
+  waitForFirstMessage(response_headers_msg);
+  ASSERT_TRUE(response_headers_msg.has_response_headers());
+  ProcessingResponse resp2;
+  auto* headers2 = resp2.mutable_response_headers();
+  auto* response_mutation = headers2->mutable_response()->mutable_header_mutation();
+  auto* add1 = response_mutation->add_set_headers();
```

Comment:
I think in general we advocate for a bit more descriptive variable names, but this being test code I'm not too bothered

Reply:
I'll keep that in mind as I write more tests.